README.md

```diff
@@ -34,47 +34,39 @@ itself in a few ways:
     their dependencies. How you install different versions of commands is
     entirely up to you.
 
-## Installation
+## Install
 
-1.  Install the `alt` binary.
+### Debian / Ubuntu / Anything using DEBs
 
-    ```sh
-    curl -sL https://github.com/dotboris/alt/raw/master/install.sh | bash -s
-    ```
-
-1.  Add the `alt` shims directory to the top of your `PATH`.
-    This lets `alt` change command versions.
-
-    For BASH:
+1.  Open the [latest release page on Github][latest-release]
+1.  Download the `.deb` file matching your system architecture
+1.  Install the `.deb` file by double clicking on it
+1.  OR Install the `.deb` file from the command line
 
     ```sh
-    echo 'export PATH="$HOME/.local/alt/shims:$PATH"' >> ~/.bashrc
-    export PATH="$HOME/.local/alt/shims:$PATH"
+    sudo apt install path/to/alt.deb
```

Comment:
```suggestion
    sudo apt install ./path/to/alt.deb
```

We're better off putting in `./` since `apt install gets real confused when you do `alt install something.deb`